autodiscern/annotations.py

```diff
@@ -0,0 +1,237 @@
+from allennlp.data.tokenizers.word_tokenizer import WordTokenizer
+from allennlp.predictors.predictor import Predictor
+from bs4 import BeautifulSoup
+from bs4.element import Tag
+import re
+from typing import Callable, Dict, List, Tuple
+from autodiscern.data_manager import DataManager
+
+
+def add_word_token_annotations(inputs: Dict[str, Dict]) -> Dict[str, Dict]:
```

Comment:
the tokenization works very well -- I looked up the [WordTokenizer](https://allenai.github.io/allennlp-docs/api/allennlp.data.tokenizers.html#word-tokenizer), it is using ` SpacyWordSplitter`. One thing I noticed is that sometimes the tokenization splits words between quotation inconsistently (i.e. `arthritis` -> [', arthritis']).  It is tricky problem because sometimes it is valid as in `friends' disease`  [friends', disease]. I think we will assess the impact of this issue once we use word embeddings

Reply:
Yes, that is tricky. I think it will come down to the type of encoding being used- word embeddings and TF/IDF will want the text stripped of all punctuation, whereas a CharNN may be able to handle the punctuation left in.

Reply:
I agree. I consider this resolved.